package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "approve-pull-request-action",
   "private": true,
-  "version": "2.0.1",
+  "version": "2.0.2",
```

could you please undo this line? I'm going to bump the version as part of my release process
```suggestion
  "version": "2.0.1",
```